packages/flutter/test/material/search_test.dart

```diff
@@ -100,11 +103,27 @@ void main() {
     await tester.tap(find.byTooltip('Search'));
     await tester.pumpAndSettle();
 
-    final TextField textField = tester.widget<TextField>(find.byType(TextField));
+    final TextField textField =
+        tester.widget<TextField>(find.byType(TextField));
     final Color hintColor = textField.decoration.hintStyle.color;
     expect(hintColor, delegate.hintTextColor);
   });
 
+  testWidgets('SearchBar color overridden', (WidgetTester tester) async {
```

Comment:
I tried to run this test with the old search code and it passes there (when I would expect it to fail), so I think you may have to re-work the test somehow so that it fails with the existing code and passes with your changes

Reply:
Done, made a test that fails without the changes